src/check.ts

```diff
@@ -41,29 +41,34 @@ async function processSig(address, safeHash) {
 }
 
 async function processSigs() {
-  console.log('Process sigs');
-  const ts = parseInt((Date.now() / 1e3).toFixed()) - delay;
-  const messages = await db.queryAsync('SELECT * FROM messages WHERE ts > ?', ts);
-  console.log('Standby', messages.length);
-  if (messages.length > 0) {
-    const safeHashes = messages.map(message => message.hash);
-    const query = {
-      sigs: {
-        __args: {
-          first: 1000,
-          where: {
-            msgHash_in: safeHashes
-          }
-        },
-        account: true,
-        msgHash: true
-      }
-    };
-    const results = await snapshot.utils.subgraphRequest(subgraphUrl, query);
-    results.sigs.forEach(sig => processSig(sig.account, sig.msgHash));
+  try {
+    console.log('Process sigs');
+    const ts = parseInt((Date.now() / 1e3).toFixed()) - delay;
+    const messages = await db.queryAsync('SELECT * FROM messages WHERE ts > ?', ts);
+    console.log('Standby', messages.length);
+    if (messages.length > 0) {
+      const safeHashes = messages.map(message => message.hash);
+      const query = {
+        sigs: {
+          __args: {
+            first: 1000,
+            where: {
+              msgHash_in: safeHashes
+            }
+          },
+          account: true,
+          msgHash: true
+        }
+      };
+      const results = await snapshot.utils.subgraphRequest(subgraphUrl, query);
```

Comment:
Instead of making a try catch of everything i would just scope it to here for the subgraphRequest and return an error with a text we can track like this console.log('Subgraph request failed', e), this is most likely what caused the issue, the other request we doing is mysql which should be stable

Reply:
Done @bonustrack